com.unity.render-pipelines.high-definition/Runtime/Lighting/AtmosphericScattering/Fog.cs

```diff
@@ -51,28 +47,50 @@ public class Fog : VolumeComponent
         /// <summary>Enable volumetric fog.</summary>
         [DisplayInfo(name = "Volumetric Fog")]
         public BoolParameter enableVolumetricFog = new BoolParameter(false);
-        /// <summary>Volumetric fog anisotropy.</summary>
-        public ClampedFloatParameter anisotropy = new ClampedFloatParameter(0.0f, -1.0f, 1.0f);
+        // Common Fog Parameters (Exponential/Volumetric)
+        /// <summary>Fog albedo.</summary>
+        public ColorParameter albedo = new ColorParameter(Color.white);
         /// <summary>Multiplier for ambient probe contribution.</summary>
         [DisplayInfo(name = "Ambient Light Probe Dimmer")]
         public ClampedFloatParameter globalLightProbeDimmer = new ClampedFloatParameter(1.0f, 0.0f, 1.0f);
-
-        /// <summary>Sets the distance (in meters) from the Camera's Near Clipping Plane to the back of the Camera's volumetric lighting buffer.</summary>
-        [Tooltip("Sets the distance (in meters) from the Camera's Near Clipping Plane to the back of the Camera's volumetric lighting buffer.")]
+        /// <summary>Sets the distance (in meters) from the Camera's Near Clipping Plane to the back of the Camera's volumetric lighting buffer. The lower the distance is, the higher the fog quality is.</summary>
         public MinFloatParameter depthExtent = new MinFloatParameter(64.0f, 0.1f);
+        /// <summary> Controls which version of the effect should be used. </summary>
```

Comment:
Controls which denoising mode to use for the volumetric effect.


Also, add a <remarks> section that says:
Reprojection mode is effective for static lighting but can lead to severe ghosting artifacts with highly dynamic lighting. Gaussian mode is effective with dynamic lighting. You can also use both modes together which produces high-quality results, but increases the resource intensity of processing the effect.